readme-parser/parse.ts

```diff
@@ -429,7 +429,13 @@ export const convertToYaml = (
 	}
 	titleWithVersion = capitalize(titleWithVersion);
 
-	const shortDescription = description.split('\n').at(0);
+	const shortDescription = description
```

Comment:
this can be undefined, can it be?

Reply:
technically it's not possible because split will always return a single-item array even if no splits happened
it's the native typing of `at()` method that causes optional chaining to be necessary. i would still prefer it over non-null assertion.